http-client/src/test/java/com/facebook/airlift/http/client/AbstractHttpClientTest.java

```diff
@@ -638,7 +639,7 @@ public void testResponseStatusMessage()
             assertNull(statusMessage);
         }
         else {
-            assertEquals(statusMessage, "message");
+            assertEquals(statusMessage, "OK");
```

Comment:
This seems unexpected given the test setup in line 629:

https://github.com/prestodb/airlift/pull/86/files#diff-b12aefec31120bfb874f675ab6f392e781b389a8acffeafe4d54b71e4202c85bR629